core-java-modules/core-java-13/pom.xml

```diff
@@ -22,24 +22,21 @@
                 <configuration>
                     <source>${maven.compiler.source.version}</source>
                     <target>${maven.compiler.target.version}</target>
-                    <release>13</release>
-                    <compilerArgs>--enable-preview</compilerArgs>
                 </configuration>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${surefire.plugin.version}</version>
-                <configuration>
-                    <argLine>--enable-preview</argLine>
-                </configuration>
             </plugin>
         </plugins>
     </build>
 
+
     <properties>
-        <maven.compiler.source.version>13</maven.compiler.source.version>
-        <maven.compiler.target.version>13</maven.compiler.target.version>
+        <java.version>17</java.version>
```

Comment:
Remove the maven compiler properties and leave only java.version

Reply:
done